trie/patriciaMerkleTrie.go

```diff
@@ -38,10 +38,8 @@ const rootDepthLevel = 0
 
 type patriciaMerkleTrie struct {
 	RootManager
+	common.TrieContext
```

Comment:
`Context` in name makes me think of `context.Context`

Reply:
not sure of a better name, but maybe something to indicate of base operations components

Reply:
I would leave it like this for now. Will change when a better name is found.